VideoWeb/VideoWeb.AcceptanceTests/appsettings.json

```diff
@@ -40,7 +43,6 @@
     "HeadlessMode": false
   },
   "QuickLinks": {
-    "JwtProviderSecret": "WNi3oF3NfduzvwUiOPlnDdUUjIlMcv7fX28ms3udpPM=",
```

Comment:
This didn't seem to match any values and doesn't seem relevant. On API this was left blank. Not sure if this should have RsaPublicKey and ValidAudience as blank. Will make sure these tests work before merging